CartApp/CartItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace CartApp
+{
+    public class CartItem 
+    {
+        public int Quantity;
```

Comment:
member field camel Casing

Reply:
I guess public field should be in PascalCase.